cf-deploy-test-teardown/cf-deploy-test-teardown.yml

```diff
@@ -1,6 +1,14 @@
 # Concourse pipeline to build deploy and teardown cf against a kube cluster
 
 resources:
+- name: src
+  type: git
+  source:
+    uri: https://github.com/SUSE/scf.git
+    branch: develop
+    paths:
+    - bin/*
```

Comment:
The only file it needs to check is `bin/dev/kube-ready-state-check.sh`, I don't see any reason to use a wildcard here.